README.md

```diff
@@ -7,49 +7,68 @@ pipeline that's more comprehensible, reproducible, and extendable.
 
 The build process for an image is described by a pipeline. Each
 [*stage*](/stages) in a pipeline is a program that, given some configuration,
-modifies a file system tree. Pipelines are defined as JSON files like this one:
+modifies a file system tree. Finally, an assembler takes a filesystem tree, and
+assembles it into an image. Pipelines are defined as JSON files like this one:
 
 ```json
 {
   "name": "Example Image",
-  "pipeline": [
+  "stages": [
     {
       "name": "io.weldr.dnf",
```

Comment:
```
[root@localhost vagrant]# osbuild -o /tmp/ 5.json

io.weldr.dnf: ea7fe2c7f659c6dbaf40045e7368a1e0837c0dfb69450f4a07549a18dcb667b9 {
  "packages": [
    "@Core",
    "httpd"
  ]
}

[/usr/lib/tmpfiles.d/journal-nocow.conf:26] Failed to resolve specifier: uninitialized /etc detected, skipping
All rules containing unresolvable specifiers will be skipped.
Failed to open file "/sys/fs/selinux/checkreqprot": Read-only file system
Traceback (most recent call last):
  File "/usr/libexec/osbuild/stages/io.weldr.dnf", line 62, in <module>
    r = main(args["tree"], args["options"])
  File "/usr/libexec/osbuild/stages/io.weldr.dnf", line 9, in main
    repos = options["repos"]
KeyError: 'repos'

io.weldr.dnf failed with code 1
```
the `dnf` stage requires more options

Reply:
Yeah, you are right. I now updated it and tested that it actually works. My initial thinking was that this example should just give the general idea, but it makes more sense to make sure it actually is a fully valid one.